.github/workflows/pr-monitor.yml

```
@@ -1,24 +1,34 @@
   pull_request:
     types: [opened, synchronize, reopened]
   pull_request_review:
-    types: [submitted]
+    types: [submitted, dismissed]
   pull_request_review_comment:
     types: [created]
+  # Resolving/unresolving a review thread is what clears (or re-raises) the
+  # threads-open signal — without this trigger, closing the LAST unresolved thread
+  # would leave a stale `pr-verdict:blocked-threads` label + sticky until some
+  # unrelated event happened to re-run the monitor (CodeRabbit PR #403).
+  pull_request_review_thread:
```

actionlint (1.7.12)

[error] unknown Webhook event "pull_request_review_thread". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)